code/game/machinery/computer/communications.dm

```diff
@@ -85,6 +85,8 @@ var/shuttle_call/shuttle_calls[0]
 			if (istype(I, /obj/item/device/pda))
 				var/obj/item/device/pda/pda = I
 				I = pda.id
+			if (istype(I,/obj/item/weapon/card/emag))

```

Comment:
Well it's in case the player tries to log in with an emag. emags are cards after all, so it kinda makes sense.